.claude/CLAUDE.MD

```
@@ -1,31 +1,43 @@
 # CLAUDE.md
 
-## Package
+## Monorepo
 
-`bria-client` — Python SDK for the Bria Engine API. Sync + async clients for image/video editing.
+SDKs for the Bria Engine API, one package per language.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)